From d591e3af402759e6ecf41e94a847184e047dabdc Mon Sep 17 00:00:00 2001
From: Xinhuang Li <buluess.li@rock-chips.com>
Date: Tue, 16 Oct 2018 14:33:48 +0800
Subject: [PATCH] ASoC: codecs: rk3328: reduce depop time to 100ms

200ms is too long which will make cts/gts fail,100ms
is enough in fact.

Change-Id: I88c75c66fcb7831afb7c8a74de2f1b3d754b84aa
Signed-off-by: Xinhuang Li <buluess.li@rock-chips.com>
---
 sound/soc/codecs/rk3328_codec.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/codecs/rk3328_codec.c b/sound/soc/codecs/rk3328_codec.c
index af1b7429b6d4..5d8784db4dc5 100644
--- a/sound/soc/codecs/rk3328_codec.c
+++ b/sound/soc/codecs/rk3328_codec.c
@@ -476,7 +476,7 @@ static int rk3328_platform_probe(struct platform_device *pdev)
 				   &rk3328->spk_depop_time);
 	if (ret < 0) {
 		dev_info(&pdev->dev, "spk_depop_time use default value.\n");
-		rk3328->spk_depop_time = 200;
+		rk3328->spk_depop_time = 100;
 	}
 
 	rk3328_analog_output(rk3328, 0);
-- 
2.35.3

